keepassxc-browser/content/autocomplete.js

```diff
@@ -335,21 +335,25 @@ class Autocomplete {
         // Extend the list's max width to input field's width or at least to the maxWidth defined in the CSS file
         this.list.style.maxWidth = `max(${Pixels(this.input.offsetWidth)}, 600px)`;
 
+        // Get body zoom radio
+	    const zoom = kpxcUI.bodyStyle.zoom || 1;
+       
         // Calculate Y offset if menu does not fit to the bottom of the screen -> show it at the top of the input field
         const menuRect = this.container.getBoundingClientRect();
         const totalHeight = menuRect.height + rect.height;
-        const menuOffset = totalHeight + rect.y > window.self.visualViewport.height ? totalHeight : 0;
+        const menuOffset = (totalHeight + rect.y) / zoom > window.self.visualViewport.height ? totalHeight/ zoom : 0;
```

Comment:
Code format. Probably there should be a space before `/`?

Reply:
fixed.